CMP1903_A1_2324/Program.cs

```diff
@@ -10,10 +10,12 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            /*
-             * Create a Game object and call its methods.
-             * Create a Testing object to verify the output and operation of the other classes.
-             */
+            Game game = new Game();
+            int sum = game.sumGame();
```

Comment:
According to the C# Google Style Guide, methods should use PascalCase, therefore it should be 'int sum = game.SumGame();'